src/parser/binary_parser.ml

```diff
@@ -619,6 +639,9 @@ let rec read_instr types input =
   | '\xC2' -> Ok (I_extend8_s S64, input)
   | '\xC3' -> Ok (I_extend16_s S64, input)
   | '\xC4' -> Ok (I64_extend32_s, input)
+  | ( '\xC5' | '\xC6' | '\xC7' | '\xC8' | '\xC9' | '\xCA' | '\xCB' | '\xCC'
+    | '\xCD' | '\xCE' | '\xCF' ) as c ->
+    Error (`Msg (Format.sprintf "illegal opcode %s" (Char.escaped c)))
```

Comment:
This can be removed as this is already handled in the same way by the default case, right ?